app/docs/dev/policies/meshtrafficpermission.md

```diff
@@ -0,0 +1,280 @@
+---
+title: MeshTrafficPermission (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficPermission](../traffic-permissions.md).
+{% endwarning %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+| ----------------- | --------- | --- | ---- |
+| Mesh              | ✅        | ✅  | ❌   |
+| MeshSubset        | ✅        | ✅  | ❌   |
+| MeshService       | ✅        | ✅  | ❌   |
+| MeshServiceSubset | ✅        | ✅  | ❌   |
```

Comment:
This seems wrong right? Isn't it supposed to be in the from column?

Reply:
Wow, good catch!